undertow/src/main/java/org/wildfly/extension/undertow/Host.java

```diff
@@ -132,13 +139,13 @@ private String getDeployedContextPath(DeploymentInfo deploymentInfo) {
 
     @Override
     public void start(StartContext context) throws StartException {
-        suspendControllerInjectedValue.getValue().registerActivity(suspendListener);
-        if(suspendControllerInjectedValue.getValue().getState() == SuspendController.State.RUNNING) {
+        suspendController.get().registerActivity(suspendListener);
+        if(suspendController.get().getState() == SuspendController.State.RUNNING) {
```

Comment:
@jasondlee fyi ^^^ is going to result in conflicts

Reply:
Noted. I'll keep an eye out so I can merge and resolve as soon as I can.